### ISSUE TITLE
KMS-6.6.1 Segmentation fault

### ISSUE THREAD

**Issue by gaelduplessix** [Kurento/bugtracker#169]
**KMS Version**:

```
Version: 6.6.1
Found modules:
	Module: 'core' version '6.6.1'
	Module: 'elements' version '6.6.1'
	Module: 'filters' version '6.6.1'
```

**Other libraries versions**:

```
ii  gstreamer1.5-libav:amd64             1.8.2.1~20160909143244.96.g493eee4.trusty  amd64        libav plugin for GStreamer
ii  gstreamer1.5-nice:amd64              0.1.13.1~20160930090742.81.geebfdab.trusty amd64        ICE library (GStreamer plugin)
ii  gstreamer1.5-plugins-bad:amd64       1.8.1.1~20160909144557.99.gf836e53.trusty  amd64        GStreamer plugins from the "bad" set
ii  gstreamer1.5-plugins-base:amd64      1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer plugins from the "base" set
ii  gstreamer1.5-plugins-good:amd64      1.8.1.1~20160909143047.112.g9ee4248.trusty amd64        GStreamer plugins from the "good" set
ii  gstreamer1.5-plugins-ugly:amd64      1.8.1.1~20160909192513.89.g2685b0f.trusty  amd64        GStreamer plugins from the "ugly" set
ii  gstreamer1.5-pulseaudio:amd64        1.8.1.1~20160909143047.112.g9ee4248.trusty amd64        GStreamer plugin for PulseAudio
ii  gstreamer1.5-x:amd64                 1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer plugins for X11 and Pango
ii  kms-core-6.0                         6.6.1.20160930110533.trusty                amd64        Kurento core module
ii  kms-elements-6.0                     6.6.1.20160930122545.trusty                amd64        Kurento elements module
ii  kms-filters-6.0                      6.6.1.20160930131150.trusty                amd64        Kurento filters module
ii  kms-jsonrpc-1.0                      1.1.2.20160930094018.trusty                amd64        Kurento jsonrpc library
ii  kmsjsoncpp                           1.6.3~20160909143252.40.gd78deb7.trusty    amd64        Kurento jsoncpp library
ii  kurento-media-server-6.0             6.6.1.20160930143513.trusty                amd64        Kurento Media Server
ii  libgstreamer-plugins-bad1.5-0:amd64  1.8.1.1~20160909144557.99.gf836e53.trusty  amd64        GStreamer development files for libraries from the "bad" set
ii  libgstreamer-plugins-base1.5-0:amd64 1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer libraries from the "base" set
ii  libgstreamer1.5-0:amd64              1.8.1.1~20160909144007.170.g0d6031b.trusty amd64        Core GStreamer libraries and elements
ii  libnice10:amd64                      0.1.13.1~20160930090742.81.geebfdab.trusty amd64        ICE library (shared library)
```

**Client libraries**
- Language: Node.js

**System description:**

KMS launched from docker image: https://github.com/Ullaakut/kurento-custom-bitrate

**What steps will reproduce the problem?**

Sorry I don't have much details here, this bug appeared a bit randomly and I don't have much time to try and reproduce it. I just figured you could make use of the logs.
Basically what I was trying to do is to build a js wrapper around RtpEndpoint, so as a result I was creating lots of elements and not cleaning everything before disconnecting. Maybe that's the reason that led to a crash.

**Please provide any additional information below.**

See attached all the logs before the segv:

[segv_logs.txt](https://github.com/Kurento/bugtracker/files/1035056/segv_logs.txt)

Hope it helps, thanks for the good work ! 👍

**Comment by j1elo** [comment]
Probably fixed. Latest Kurento version: 6.14
